doc/how_nml_works.md

```diff
@@ -1,6 +1,6 @@
 # Going Into Detail: How NeosModLoader Interfaces with Neos
 
-NeosModLoader only interfaces with Neos in two places: the hook and the compatibility hash.
+NeosModLoader with Neos in two main places: the hook and the compatibility hash.
```

Comment:
Seems to be missing 'interfaces with'

Reply:
Fixed in [be2481e](https://github.com/neos-modding-group/NeosModLoader/pull/54/commits/be2481e164be748596ab0b66ea6a63dcbd29bea3)